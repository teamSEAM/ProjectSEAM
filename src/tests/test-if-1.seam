entity World:
	func start():
		screen.init(100,100)
		if(1==1):
			screen.out("1==1 OK")
		else:
			screen.out("1==1 FAIL")
