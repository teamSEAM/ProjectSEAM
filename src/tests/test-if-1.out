1==1 OK
